src/backend/distributed/citus--8.2-1--8.2-2.sql

```diff
@@ -0,0 +1,3 @@
+/* citus--8.2-1--8.2-2.sql */
+
+DROP FUNCTION IF EXISTS pg_catalog.create_insert_proxy_for_table(regclass,regclass);
```

Comment:
Today I realized that we still have this UDF and the tests. Happy to see that we're removing it within an open PR 😄